components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/internal/handler/LoginEventHookHandler.java

```diff
@@ -163,6 +163,13 @@ private void handleEventForProfile(Event event, EventData eventData, EventProfil
             return;
         }
 
+        if (EventHookHandlerUtils.isB2BUserLogin(eventData.getAuthenticationContext())) {
+            log.debug(
```

Comment:
Lets add a isDebugEnabled check here

Reply:
Since we don't do any string manipulation, is debug enabled not needed i guess

Reply:
fixed with https://github.com/wso2-extensions/identity-webhook-event-handlers/pull/81/commits/bd85fb281b48d290592bb194ed1251d2600e28bb